include/odp/api/spec/schedule.h

```diff
@@ -187,6 +187,51 @@ void odp_schedule_prefetch(int num);
  */
 int odp_schedule_num_prio(void);
 
+/**
+ * Initialize schedule configuration options
+ *
+ * Initialize an odp_schedule_config_t to its default values.
+ *
+ * @param[out] config  Pointer to schedule configuration structure
+ */
+void odp_schedule_config_init(odp_schedule_config_t *config);
+
+/**
+ * Global schedule configuration
+ *
+ * Initialize and configure scheduler with global configuration options
+ * to schedule events across different schedule queues. The scheduler module
+ * should be started before creating any odp queues. This should happen before
+ * an application itself or any other ODP module creates any schedued queue.
+ * An application can pass NULL value to use default configuration.
+ *
+ * The initialization sequeunce should be,
+ * odp_schedule_capability()
+ * odp_schedule_config_init()
+ * odp_schedule_config()
+ * odp_schedule()
+ *
+ * @param config   Pointer to scheduler configuration structure
```

Comment:
, or NULL for default configuration.

Reply:
NULL value specification is still needed (for @param config): "..., or NULL for default configuration."